src/routers/controllers/confirmCompany.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response, NextFunction } from "express";
+import { ConfirmCorrectCompany } from "../handlers/yourCompanies/confirmCorrectCompany";
+import * as constants from "../../constants";
+import { Session } from "@companieshouse/node-session-handler";
+import { getLoggedInUserEmail } from "../../lib/utils/sessionUtils";
+import { isCompanyAssociatedWithUser } from "../../services/userCompanyAssociationService";
+import * as urlUtils from "../../lib/utils/urlUtils";
+
+export const confirmCompanyGet = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+        const session: Session = req.session as Session;
+        const companyProfile = session.data.extra_data.companyProfile;
+        const viewData = await new ConfirmCorrectCompany().execute(req.t, companyProfile);
+        res.render(constants.CONFIRM_COMPANY_PAGE_TEMPLATE, viewData);
+    } catch (e) {
+        return next(e);
```

Comment:
Test coverage indicates that this is not tested

Reply:
As above